docs/guides/specify-runtime-options.mdx

```diff
@@ -0,0 +1,384 @@
+
+<Tabs>
+  <TabItem value="primitivesV2" label="V2 Primitives">
+Shots (or precision) can be specified in multiple places in V2.  They are prioritized as follows:
+
+For any Sampler PUB:
+
+1. Integer-valued shots contained in the PUB
+2. The `run(...,shots=val)` value
+3. The `options.default_shots` value
+
+For any Estimator PUB:
+
+1. Float-valued precision contained in the PUB
+2. The `run(...,precision=val)` value
+3. The `options.default_shots` value
+4. The `options.default_precision` value
+
+Example:
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+# Setting shots during primitive initialization
+sampler = Sampler(mode=backend, options={"default_shots": 4096})
+
+# Setting options after primitive initialization
+# This uses auto complete.
+sampler.options.default_shots=2000
+
+# This does bulk update.  The value for default_shots is overridden if you specify shots with run() or in the PUB.
+sampler.options.update(default_shots=1024, dynamical_decoupling={"sequence_type": "XpXm"})
+
+# Sample two circuits at 128 shots each.
+sampler.run([circuit1, circuit2], shots=128)
+ ```
+  </TabItem>
+
+  <TabItem value="v1primitives" label="V1 Primitives">
+Previously, shots could be set during the call to `backend.run()`. For example, `backend.run(shots=1024)`. Now, that setting is part of the execution
+options ("second level option"). This can be done during the primitive setup:
+
+```python
+from qiskit_ibm_runtime import Estimator, Options
+
+options = Options()
+options.execution.shots = 1024
+
+estimator = Estimator(session=backend, options=options)
+```
+
+If you need to modify the number of shots set between iterations (primitive calls), you can set the
+shots directly in the `run()` method. This overwrites the initial `shots` setting.
+
+```python
+from qiskit_ibm_runtime import Estimator
+
+
+estimator = Estimator(session=backend)
+
+estimator.run(circuits=circuits, observables=observables, shots=50)
+
+# other logic
+
+estimator.run(circuits=circuits, observables=observables, shots=100)
+```
+
+For more information about the primitive options, refer to the
+[Options class API reference](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.Options).
+</TabItem>
+</Tabs>
+
+### Maximum execution time
+
+The maximum execution time (`max_execution_time`) limits how long a job can run.  If a job exceeds this time limit, it is forcibly cancelled.  This value applies to single jobs, whether they are run in job, session, or batch mode.
+
+The value is set in seconds, based on quantum time (not wall clock time), which is the amount of time that the QPU is dedicated to processing your job.  It is ignored when using local testing mode because that mode does not have quantum time.
```

Comment:
```suggestion
The value is set in seconds, based on quantum time (not wall clock time), which is the amount of time that the QPU is dedicated to processing your job.  It is ignored when using local testing mode because that mode does not use quantum time.
```
or "involve", maybe?